src/model/SVGFile.java

```diff
@@ -34,21 +42,34 @@ public void setXmlcontent(String xmlcontent) {
 		this.xmlcontent = xmlcontent;
 	}
 
+	
+	/*
+	 * save SVG file into files
+	 */
 	@Override
-	public boolean writeSVGFile(String content) {
-		// TODO Auto-generated method stub
-		return false;
+	public boolean writeSVGFile() {
+		file = new File(String.format(filepath, this.getFilnename()));
```

Comment:
comment plz, what is the var filepath ? and what does File(String.format) and the format, why ?